src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,44 @@
 
 class Animal {
   // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+  }
 }
```

Comment:
```suggestion
class Animal {
  // write your code here
  constructor(name) {
    this.name = name;
    this.health = 100;
    
    Animal.alive.push(this);
  }
}
```
Herbivores and carnivores are also animals (they extend from Animal class), so there is no need to duplicate the same code in the child classes.